### ISSUE TITLE
Several second page load when 100+ games added 

### ISSUE THREAD

**Issue by ergoxi** [sbondCo/Watcharr#665]
Commenting this line out fixes it for me 
https://github.com/sbondCo/Watcharr/blob/d21eb066009aa4ab5e6be7acfda58f5e3eabc88d/src/lib/poster/GamePoster.svelte#L100

**Comment by IRHM** [comment]
Hi @ergoxi, thank you for reporting this issue. I will get on it soon.

I haven't got that many games on my list, but I will definitely have to test this on a dev server.

I think it might be time for me to implement infinite loading (pagination) on the watched list page so that it isn't trying to decode all them blurhashes at once.

Related #300

**Comment by bretthysuik** [comment]
I don't think this is limited to games. I imported ~10 years of movies and tv shows from Trakt and it takes approx 8 seconds to load the main page (the call to `/api/watched` alone takes ~3 seconds because it's returning *everything*)

**Comment by IRHM** [comment]
Just wanted to provide an update; still actively working on this. I decided to go the pagination route because theres no other way around that 3s load mentioned, but a happy side effect should be that it fixes the game poster concern too (will still have to test that though, 40 page size might be of concern here if all of them are games).

Just taking me so long because pagination requires a lot of reworking of the app (the main list being the easy part, but now everywhere we use the watched list data including filters/sort is being moved server side).

**Comment by IRHM** [comment]
I couldn't help but refactor the entire server so this took ages, but its done in the 3.0 branch now.

Edit: Please let me know if anyone is still experiencing issues after updating!
